Authenticators/LDAP/LDAPauth.py

```diff
@@ -514,7 +521,7 @@ def authenticate(self, name, pw, certlist, certhash, strong, current = None):
                 
             # Parse the user information.
             uid = int(match[1][cfg.ldap.number_attr][0])
-            displayName = match[1][cfg.ldap.display_attr][0]
+            displayName = match[1][cfg.ldap.display_attr][0].decode()
```

Comment:
What does this decode from and into? What’s the context to this?

Reply:
ldap returns bytes, we need a string

Reply:
shouldn't we specify a code-page that should be used for decoding?

Reply:
We could, but it defaults to utf-8 and that will work with any modern ldap server without a very strange config.

Reply:
Okay then :)